DOL.WHD.Section14c.Web/src/modules/services/authService.js

```diff
@@ -56,20 +56,23 @@ module.exports = function(ngModule) {
           stateService.user = data;
           if (data.organizations.length > 0) {
             stateService.ein = data.organizations[0].ein; //TODO: Add EIN selection?
+          } else {
+            d.resolve();
           }
-          if (!stateService.isAdmin) {
-            stateService.loadSavedApplication().then(
-              function() {
-                // start auto-save
-                if (stateService.ein) {
-                  autoSaveService.start();
-                }
-                d.resolve();
-              },
-              function(error) {
-                d.reject(error);
-              }
-            );
+          if (!stateService.IsPointOfContact) {
+            d.resolve();
+            // stateService.loadSavedApplication().then(
+            //   function() {
+            //     // start auto-save
+            //     if (stateService.ein) {
+            //       autoSaveService.start();
+            //     }
+            //     d.resolve();
+            //   },
+            //   function(error) {
+            //     d.reject(error);
+            //   }
+            // );
```

Comment:
A big wad of commented code.  I know this PR is still a WIP, but just flagging it for the future.

Reply:
removed